packages/deb/hazelcast/DEBIAN/control

```diff
@@ -1,6 +1,6 @@
 Package: ${HZ_DISTRIBUTION}
 Source: hazelcast
-Version: ${PACKAGE_VERSION}
+Version: 1:${PACKAGE_VERSION}
```

Comment:
Adding the epoch field means that we have to add use it when installing a specific version:
```diff
- sudo apt install hazelcast=5.2-SNAPSHOT
+ sudo apt install hazelcast=1:5.2-SNAPSHOT
```

This means the change is much more intrusive than expected. I'm having second thoughts about it, maybe we should leave it as it is. 

What do you think @frant-hartm?

Reply:
Good to see this failing in the tests :-)

Yeah, I agree that it is too intrusive, for little value - not many people have installed any 5.2021.xx package.

We need to handle this somehow, probably best is a note in the documentation advising to uninstall hazelcast first if you have a version using the older older scheme

Reply:
> Good to see this failing in the tests :-)

That was my first thought as well 💪 

Added new PR with additional description: https://github.com/hazelcast/hazelcast-packaging/pull/141